containers/build_backend.go

```diff
@@ -44,10 +44,10 @@ type CompileBackendOpts struct {
 }
 
 func goBuildImage(distro executil.Distribution, opts *executil.GoBuildOpts, goVersion string) string {
-	os, arch := executil.OSAndArch(distro)
+	os, _ := executil.OSAndArch(distro)
 
 	// For arm/v7 and arm/v6 we want to use viceroy
```

Comment:
Is this comment still valid now that the condition has changed?

Reply:
true it is not!